es-app/src/guis/GuiMenu.cpp

```diff
@@ -373,6 +377,11 @@ void GuiMenu::openScreensaverOptions() {
 	mWindow->pushGui(ggf);
 }
 
+void GuiMenu::openVirtualSystemSettings() {
+	GuiVirtualSystemsOptions* ggf = new GuiVirtualSystemsOptions(mWindow);
+	mWindow->pushGui(ggf);
```

Comment:
LGTM :) Actually, it was done like this for consistency with previous options, but you're right - ggf is a pure copy and paste of the previous method :)

I can certainly consolidate it in a single line.